src/routes/exportData.ts

```diff
@@ -138,32 +146,40 @@ const createAnswersCSVFile = (
   csvWriter.writeRecords(gapAnswers);
 };
 
-const transformSegmentAnswersToGapAnswers = (
-  answers: SegmentAnswer[],
-  language: string
-): GapAnswer[] => {
+const removeDuplicateAnswers = (answers: SegmentAnswer[]): SegmentAnswer[] => {
   const answersGroupedBySegmentId = groupBy<SegmentAnswer>(
     answers,
     'segmentId'
   );
-  const segmentGroups: SegmentAnswer[][] = Object.values(
+  const segmentGroupsBySegmentId: SegmentAnswer[][] = Object.values(
```

Comment:
It's probably not necessary to create another variable you could just wrap line 50 in the `Object.values` if you wanted?

Reply:
definitely, would also make naming things easier 😀